### PR TITLE
[CCTV] Added New CCTV System that will Record Crime

### PR DESCRIPTION
**Pull request by maxgestic**
# Features
- Adds CCTV cameras all over the city that record crime and are repayable by police
- Police Evidence Storage for CCTV at MRPD
- Recordings can be passed between players using /recordings
- Cameras can be shot out
- Police can access cameras by standing near a camera and then pressing A when it pops up

ADD SQL

```sql
CREATE TABLE `camera_recordings` (
  `id` varchar(8) NOT NULL,
  `groupId` int(11) NOT NULL,
  `stored` varchar(64) NOT NULL,
  `recordedAt` timestamp NOT NULL DEFAULT current_timestamp(),
  UNIQUE KEY `id` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
```

# Demo Videos
https://www.youtube.com/watch?v=-wjZDYpMQzY&ab_channel=SirChainsmokerGollum
https://www.youtube.com/watch?v=s3_AXmeSxOw&ab_channel=SirChainsmokerGollum

### CODE REVIEW

File: resources/banking/server.lua [hunk 1]
```diff
@@ -1,7 +1,7 @@
 local robbedATMs = {}
 local players_that_blew_atm = {}
 local ATM_ROBBED_TIMEOUT_HOURS = 4
-local NUM_COPS_REQUIRED = 2
+local NUM_COPS_REQUIRED = 0
```
Comment (minipunch) [inline resources/banking/server.lua:4]: 👁️

Comment (maxgestic) [reply resources/banking/server.lua]: Fixed

**Comment by minipunch** [comment]
Can u please explain how it works

Like any commands, how we access the cameras, etc

**Comment by maxgestic** [comment]
> Can u please explain how it works
> 
> Like any commands, how we access the cameras, etc
> 
> Is it just all done through the `/recordings` command ?

Added it to description you can access a camera network by standing near the camera, directly underneath works the best. Popup will show saying like Press E to open camera and then players can either watch a recording or "take" it, this recording will be off all camera angles on the network, so for example all cameras at alta

**Comment by minipunch** [comment]
guessing there is an .sql file or something I need to run .. ?

**Review by minipunch — APPROVED** [review]

**Pull request merged**